### DISCUSSION TITLE
single repo, deploy multiple vercel projects

### DISCUSSION THREAD

**Discussion by skene** [vercel/vercel#4452] [Help] [answered]
I noticed that `--name` flat for vercel is deprecated which allows for deployment to several vercel projects from a single repo. Will there be a way to do this once the `--name` has been removed in the future?

Thanks

**Comment by coetry** [comment, marked as answer]
Hi @skene ! We're working really hard to enable monorepo support on our dashboard which will allow you to create many Vercel projects with their own configuration from a single repo. We hope to release that as soon as possible :smile: .
